daisy_integration_tests/v2v_adapt_e2e.test.gotmpl

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
I expect to see a lot of v2v tests (>30, let me know if it's not the case), so we can consider 2 things:
1. put them in a subfolder
2. estimate and increase the quota of the test proj, or find a way to limit the parallel num

Reply:
The idea is to reuse existing translate tests by providing translate workflow as a variable. The only new files will be the actual adapt/translate workflows, which, so far, have only a couple of variants. If this grows, I'll consider extracting them outside the tests folder.

Regarding the parallel test execution. I've reduced the parallel count to 10. The frequency is currently 12hrs so this should add 12.5% resource use compared to the existing translate tests which run every 3hr and have parallel count of 30. Note that there are other tests so the real resource add is probably half of 12.5% (guestimate).